pkg/dataloaders/cedar_system_linked_system_intakes.go

```diff
@@ -0,0 +1,49 @@
+package dataloaders
+
+import (
+	"context"
+	"errors"
+
+	"github.com/cms-enterprise/easi-app/pkg/models"
+)
+
+func (d *dataReader) batchCedarSystemLinkedSystemIntakes(ctx context.Context, requests []models.SystemIntakesByCedarSystemIDsRequest) ([][]*models.SystemIntake, []error) {
+	data, err := d.db.SystemIntakesByCedarSystemIDs(ctx, requests)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	ids := make([]string, len(requests))
+	for i := range requests {
+		ids[i] = requests[i].CedarSystemID
+	}
```

Comment:
fwiw apparently they're going to add `slices.Repeat` in the [near future](https://github.com/golang/go/issues/65238).

Reply:
This code, though, is not repeating the same element like `slices.Repeat` would be, right? This is more akin to a `.map`, pulling the `CedarSystemID` out of each request into a slice of IDs

Reply:
Oh, you're right. For some reason I read this as it was making a slice of all the same CEDAR system IDs.